### PR TITLE
open api specification for tezedge

### PR DESCRIPTION
**Pull request by mambisi**
This PR add open api specification to tezedge

### CODE REVIEW

File: spec/florence.yml [hunk 1]
```diff
@@ -0,0 +1,10890 @@
+openapi: 3.0.0
+info:
+  title: Tezedge Protocol RPCs
+  termsOfService: https://www.viablesystems.io/
+  contact:
+    name: API Support
+    url: https://github.com/tezedge/tezedge
+    email: support@example.com
```
Comment (bkontur) [inline spec/florence.yml:8]: @mambisi @jurajselep 
Mambisi, please put here some correct email

**Comment by mambisi** [comment]
Which email though? @jurajselep

File: spec/florence.yml [hunk 2]
```diff
@@ -0,0 +1,10890 @@
...
+  license:
+    name: MIT License
+    url: https://github.com/tezedge/tezedge/blob/master/LICENSE
+  version: 1.5.0
```
Comment (bkontur) [inline spec/florence.yml:12]: @mambisi 
put here please: `v1.6.6`

File: spec/tezedge-openapi.json [hunk 3]
```diff
@@ -0,0 +1,5293 @@
+{
+  "openapi": "3.0.0",
+  "info": {
```
Comment (bkontur) [inline spec/tezedge-openapi.json:3]: @mambisi @jurajselep 
please, make sure that information are correct and relevant (email, links) and not like `This is a sample server for a pet store.`

and copy the same information to all files

**Comment by bkontur** [comment]
@mambisi @jurajselep @directcuteo 
and I would better move that `spec` directory to the rpc: `rpc/spec`,
I dont know how explorer is reading that, but we should expose this files throught RPC,
like:
`http://localhost:18732/openapi/tezedge-openapi.json`

**Review by bkontur — CHANGES_REQUESTED** [review]
@mambisi please PR comments

File: spec/florence.yml (hunk 2 above)
Comment (mambisi) [reply spec/florence.yml:12]: Ok

**Review by bkontur — APPROVED** [review]

**Pull request merged**